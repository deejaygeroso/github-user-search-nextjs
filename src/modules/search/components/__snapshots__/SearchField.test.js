import React from 'react';
import SearchField from '../SearchField';
import renderer from "react-test-renderer";

it("renders correctly", () => {
    const tree = renderer
    .create(
        <SearchField onChange={()=>{}} onSubmit={()=>{}}/>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
})