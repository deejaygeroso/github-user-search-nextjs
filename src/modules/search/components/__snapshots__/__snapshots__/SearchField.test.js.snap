// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="MuiPaper-root-4 MuiPaper-elevation1-7 MuiPaper-rounded-5 SearchField-root-1"
>
  <div
    className="MuiInputBase-root-31 SearchField-input-2"
    onClick={[Function]}
    onKeyPress={[Function]}
  >
    <input
      className="MuiInputBase-input-41"
      onBlur={[Function]}
      onChange={[Function]}
      onFocus={[Function]}
      placeholder="Search by username"
      type="text"
    />
  </div>
  <button
    aria-label="Search"
    className="MuiButtonBase-root-54 MuiIconButton-root-48 SearchField-iconButton-3"
    disabled={false}
    onBlur={[Function]}
    onContextMenu={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
    tabIndex="0"
    type="button"
  >
    <span
      className="MuiIconButton-label-53"
    >
      <svg
        aria-hidden="true"
        className="MuiSvgIcon-root-57"
        focusable="false"
        role="presentation"
        viewBox="0 0 24 24"
      >
        <path
          d="M15.5 14h-.79l-.28-.27C15.41 12.59 16 11.11 16 9.5 16 5.91 13.09 3 9.5 3S3 5.91 3 9.5 5.91 16 9.5 16c1.61 0 3.09-.59 4.23-1.57l.27.28v.79l5 4.99L20.49 19l-4.99-5zm-6 0C7.01 14 5 11.99 5 9.5S7.01 5 9.5 5 14 7.01 14 9.5 11.99 14 9.5 14z"
        />
        <path
          d="M0 0h24v24H0z"
          fill="none"
        />
      </svg>
    </span>
    <span
      className="MuiTouchRipple-root-66"
    />
  </button>
</div>
`;
